ocs_ci/ocs/scale_lib.py

```diff
@@ -808,6 +820,60 @@ def check_and_add_enough_worker(worker_count):
                 )
             return True
 
+        # Add enough worker for RHV
+        elif (
```

Comment:
There are a lot of if..else block, would this be more elegant? 
https://stackoverflow.com/questions/17881409/whats-an-alternative-to-if-elif-statements-in-python
wdyt? (But, of course this can be taken into a subsequent PR)

Reply:
We have couple of function which needs enhancement or update to reduce the code duplication and to avoid lot of if conditions, for now no plans to change this. Will plan by 4.9, backlog jira card https://projects.engineering.redhat.com/browse/OCSQE-657